docs/kratos/social-signin/95_account-linking.mdx

```diff
@@ -46,15 +46,27 @@ To try out account un-linking, use the Ory Account Experience. Follow these step
 
 ## Automatic account linking
 
-While it is convenient for users, automatic account linking creates an attack vector that can allow malicious actors to steal user
-accounts.
-
-:::warning
+:::info
 
-Since it constitutes a security threat, automatic account linking is not available in Ory.
+Automatic account linking was previously not available in Ory. It has been recently added and is now enabled on the Ory Network as
+well as on the open-source master branch.
 
 :::
```

Comment:
I think we can just remove this disclaimer

Reply:
Fine by me. I wanted to leave it in because it was not possible for a long time.

Reply:
I agree we dont need the disclaimer here, we can point this out in the changelog/newsletter/community/etc.